### ISSUE TITLE
Segmentation fault      (core dumped)

### ISSUE THREAD

**Issue by tedljw** [lonePatient/albert_pytorch#15]
你好，我在按步骤执行到sh run_classifier_lcqmc.sh 时，运行出现
run_classifier_lcqmc.sh: line 22: 33607 Segmentation fault ，
我的环境是1080ti的显卡，pytorch 1.1 

error前出现的打印为：
11/01/2019 16:48:38 - INFO - root -   ***** Running training *****
11/01/2019 16:48:38 - INFO - root -     Num examples = 238766
11/01/2019 16:48:38 - INFO - root -     Num Epochs = 5
11/01/2019 16:48:38 - INFO - root -     Instantaneous batch size per GPU = 64
11/01/2019 16:48:38 - INFO - root -     Total train batch size (w. parallel, distributed & accumulation) = 64
11/01/2019 16:48:38 - INFO - root -     Gradient Accumulation steps = 1
11/01/2019 16:48:38 - INFO - root -     Total optimization steps = 18655

我debug的过程中发现是在：outputs = model(**inputs) ， run_classifier.py：115的位置。

我不使用显卡，在cpu环境中不会出现问题。

**Comment by lonePatient** [comment]
@tedljw 这个问题没遇到过，感觉你的pytorch-gpu环境没配好，之前pytorch跑其他模型是否会出现问题呢？或者pytorch版本问题。回头我试下1.1版本看看

**Comment by tedljw** [comment]
> @tedljw 这个问题没遇到过，感觉你的pytorch-gpu环境没配好，之前pytorch跑其他模型是否会出现问题呢？或者pytorch版本问题。回头我试下1.1版本看看

你好，非常感谢你的回答。
首先，我看到你readme上面写的pytorch是1.1的版本。所以我也配了1.1的pytorch，同时我的cuda也是9.0的。
然后，我在pytorch1.1 gpu的环境上其他的模型也都跑过，没有出现过问题，基于hughugface的代码，我也funtune过bert，也没出现过问题。
请问大佬你用的pytorch是什么版本的，我改一下环境再试一下。

**Comment by tedljw** [comment]
> @tedljw 这个问题没遇到过，感觉你的pytorch-gpu环境没配好，之前pytorch跑其他模型是否会出现问题呢？或者pytorch版本问题。回头我试下1.1版本看看

你好，我debug到，出现Segmentation fault在modeling_albert.py的162行：
embeddings = self.LayerNorm(embeddings)
传入的embeddings维度是64*64*312

**Comment by tedljw** [comment]
64 * 64 * 312

**Comment by lonePatient** [comment]
@tedljw 问题还发生吗？我昨晚测试了下是没问题的，我的环境是ubuntu18.04 python3.7 cuda9.0 pytorch1.1 cudann7.5

**Comment by tedljw** [comment]
> @tedljw 问题还发生吗？我昨晚测试了下是没问题的，我的环境是ubuntu18.04 python3.7 cuda9.0 pytorch1.1 cudann7.5

我把python提升到3.7后，没问题了。我之前的py是3.6.
